test/rest.middleware.test.js

```diff
@@ -164,6 +164,40 @@ describe('loopback.rest', function() {
     }, done);
   });
 
+  it('should report 200 for legacy explorer route /routes', function(done) {
+    app.model(MyModel);
```

Comment:
It wasn't necessary, I just wanted to ensure there would be values returned inside the responses. By removing it though I can improve the test a bit and look for `[]` and `{}` for the `/routes` and `/models` response bodies respectively.